enclave-tls/src/attesters/internal/etls_enclave_attester_load_all.c

```diff
@@ -16,6 +18,11 @@
   #define PATTERN_SUFFIX ".so"
 #endif
 // clang-format on
+#ifdef SGX
+#include <sgx_error.h>
```

Comment:
move `#include <dirent.h>` to here.

Reply:
It's not define SGX, dirent will be used in occlum and host mode.
#ifndef SGX
#include <dirent.h>
#endif